docs/products/kafka/howto/kafka-streams-with-aiven-for-kafka.rst

```diff
@@ -0,0 +1,217 @@
+Kafka Streams with Aiven for Kafka
+==================================
+
+Kafka streams and streams API allows for streaming data through the
+heart of Kafka, the brokers. The data that passes through the brokers
+has a key-value structure. As this data enters the brokers, it is
+serialized by the streams API to a byte array, and as it exits the
+brokers it is de-serialized back to the original key-value structure,
+again by the streams API. Kafka streams also allows for transformation
+of data that enters the brokers in real-time, and feed the transformed
+data back into Kafka as another, transformed stream of data. This makes
+Kafka streams quite a powerful concept in terms of the many use cases it
+can address in the world of real-time data processing and analysis.
+
+This help article explores how to realize Kafka streams with Aiven for
+Kafka and Schema registry.
+
+**Note:** As of version 3.0, Aiven for Apache Kafka no longer supports
+Confluent Schema Registry. For more information, see `this article that
+describes the replacement,
+Karapace <https://help.aiven.io/en/articles/5651983>`__ .
+
+The article is divided into the following parts:
+
+-  Creating an Aiven for Kafka service
+
+-  Modifying the KafkaMusicExample application to work with Aiven for
+   Kafka and Schema registry
+
+Creating an Aiven for Kafka service is documented in the “ `Getting
+started with Aiven for
+Kafka <https://help.aiven.io/kafka/getting-started-with-aiven-kafka>`__
+” help article. After creating the Kafka service, please create topics
+“play-events” and “song-feed” via the Aiven console.
+
+The following information will be required in the configuration process:
+
+-  Kafka service's service URL
+
+.. image:: kafka-services-service-url.jpg
```

Comment:
Every image needs an `alt` tag. The parser doesn't pick this up but does fail horribly on the rest of the file if it's not there! Try adding alt tags to fix the build errors (sorry).

Please put the images in the `images/` directory (the structure reflects the content structure, makes it easier to handle the content types at scale). Thanks!